src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 20px;
+    height: 16px;
+    background-image: url("../images/star.svg");
+    background-repeat: no-repeat;
+    background-origin: border-box;
+    background-position: 2px center;
+  }
```

Comment:
As for me it looks strange. If our stars became bigger, we have to remember, that we added a half of difference to our background-position. I think it would be better to use margin-right.